plugins/outputs/honeycomb/README.md

```diff
@@ -0,0 +1,34 @@
+# Honeycomb Output Plugin
+
+This plugin writes data to [Honeycomb.io](https://honeycomb.io).
+
+
+### Configuration
+
+```toml
+  ## Honeycomb authentication token
+  apiKey = "API_KEY"
+
+  ## Dataset name in Honeycomb to send data to
+  dataset = "my-dataset"  
+
+  ## Special tags that will not get prefixed by the measurement name
+  ## This should be set if you specified global tags, and it should include the list of all global tags + host
+  ## Default value for this list is: host
+  #specialTags = ["host"]
+  
+  ## Optional: the hostname for the Honeycomb API server
+  #apiHost = "https://api.honeycomb.io/""
+```
```

Comment:
Please use snake_case in TOML.
```suggestion
```toml
  ## Honeycomb authentication token
  api_key = "API_KEY"

  ## Dataset name in Honeycomb to send data to
  dataset = "my-dataset"  

  ## Special tags that will not get prefixed by the measurement name
  ## This should be set if you specified global tags, and it should include the list of all global tags + host
  ## Default value for this list is: host
  #non_prefixed_tags = ["host"]
  
  ## Optional: the hostname for the Honeycomb API server
  #url = "https://api.honeycomb.io/""
```
```